cmd/minikube/cmd/start.go

```diff
@@ -822,7 +821,7 @@ func validateFlags(cmd *cobra.Command, drvName string) {
 		}
 
 		// conntrack is required starting with kubernetes 1.18, include the release candidates for completion
-		version, _ := util.ParseKubernetesVersion(getKubernetesVersion(nil))
+		version, _ := pkgutil.ParseKubernetesVersion(getKubernetesVersion(nil))
```

Comment:
I would say keep the util and get rid of the pkgutil

Reply:
Change as suggested, thanks.